src/dbus_api/util.rs

```diff
@@ -35,7 +34,7 @@ pub fn get_next_arg<'a, T>(iter: &mut Iter<'a>, loc: u16) -> Result<T, MethodErr
 
 
 /// Translates an engine error to a dbus error.
-pub fn engine_to_dbus_err(err: &EngineError) -> (DbusErrorEnum, String) {
+pub fn engine_to_dbus_err(err: &EngineError) -> (u16, String) {
```

Comment:
Header comment needs a slight update.

Reply:
ok. I also am going to rename to `engine_to_dbus_err_tuple` for more clarity.